gradle/dependencies.gradle

```diff
@@ -12,6 +12,7 @@ ext.versions = [
     testhelp: '0.4.0',
     sqljs: '1.0.0',
     paging: '2.1.2',
+    paging3: '3.0.0-beta02',
```

Comment:
We probably shouldn't ship this until 3.0.0 stable, but we can still land it and ship snapshots for now to iterate on.

Reply:
What's the reasoning here?

Reply:
An API change or behavior change could break the implementation. Beta is _mostly_ API stable but definitely not behaviorally stable. When we make a release all of the versions indicate that they are stable which is a promise we may not be able to keep with an unstable dependency. If we want a release, we can exclude it from the main one and do a special one-off that only has paging with our own 'beta' tag.